projects/go-lib/src/lib/components/go-header/go-header.component.html

```diff
@@ -0,0 +1,20 @@
+<header class="go-header">
+  <div class="go-header__left"
+       [ngClass]="{ 'go-header__left--collapsed': !sideNavService.navOpen }">
+    <go-icon class="go-header__menu"
+             icon="menu" 
+             (click)="toggleSideMenu()">
+    </go-icon>
+    <div class="go-header__logo-container"
+         [ngClass]="{ 'go-header__logo-container--collapsed': !sideNavService.navOpen }">
+      <img class="go-header__logo" [src]="logo"/>
```

Comment:
Should we have an alt text here on this image? What happens if an invalid src or no src gets passed in?

Reply:
Good point, I added a binding and output the `alt` here. :+1: